tests/hostnameshardtests/l7_hostname_shard_test.go

```diff
@@ -114,6 +114,8 @@ func AddConfigMap() {
 			Name:      "avi-k8s-config",
 		},
 	}
+	aviCM.Data = make(map[string]string)
+	aviCM.Data["logLevel"] = "DEBUG"
```

Comment:
is this needed?

Reply:
Not needed. Can be plugged in, on need basis. Removed it.